CONTRIBUTING.md

```diff
@@ -35,7 +35,8 @@ We use the Squash and Merge model, which means that all commits in a Pull Reques
 Pull requests can be merged by members of the [the repository maintainers' team](https://github.com/eiffel-community/community/blob/master/CONTACT.md). There is a certain protocol to adhere to, however, as well as expectations on membership.
 1. All maintainers are expected to make the effort to participate in the review of Pull Requests. Every maintainer may not review everything in detail, but everyone can make the effort to chime in on some. Remember that expedient high quality reviews are crucial to the long term survival of any open source project.
 1. All community members, maintainers or not, are strongly encouraged to participate in reviews even if they do not feel entirely qualified to assess the pull request. Looking at changes and participating in review discussions is one of the best ways to learn, and presents an excellent opportunity to ask questions. And remember, participating in a review is not the same as having to make the final decision.
-1. A Pull Request should be approved by at least two maintainers (including the one doing the merging). For this to function well, the above point on participation is critical.
+1. For a non graduated project we recommend at least two maintainers (including the one doing the merging) to approve the Pull Request. For this to function well, the maintainers need to participate as stated in the previous point.
```

Comment:
Should we perhaps be explicit about what "non graduated" means, i.e. say "sandbox" instead? People not knowing the project lifecycle docs by heart would otherwise go "and what the heck does 'non graduated' mean?".

Reply:
Good point. Fixed in 53871bd